src/index.js

```diff
@@ -94,29 +123,12 @@ function makeConfig () {
  * @throws            An error if the JSON could not be parsed
  */
 function loadJson (path) {
-  const text = Fs.readFileSync(path, 'utf8')
-  if (text) {
-    try {
-      return JSON.parse(text)
-    }
-    catch (err) {
-      require('colors')
-      const file = Path.basename(path)
-      const message = (
-        '\n  Error! ' + err.message.replace('JSON', `"${file}"`)
-        + '\n\n  Edit the file to fix this, then try again'
-        + '\n').red
-
-      // CLI
-      if (global['ALIAS_CLI']) {
-        console.warn(message)
-        process.exit(0)
-      }
-
-      // API
-      console.warn(`\n  [Alias HQ]\n${message}\n`.red)
-      throw(err)
-    }
+  const text = Fs.readFileSync(path, 'utf8').toString()
+  const {config, error} = ts.parseConfigFileTextToJson(path, text)
```

Comment:
I haven't added any linting to this project yet (will do that). I use Standard JS with Stroustrop brackets, so:

- spaces round deconstruction braces please
- else on newline